CHANGES.md

```diff
@@ -138,25 +133,35 @@ OpenSSL 3.0
    *Kurt Roeckx*
 
  * The command line utilities dhparam, dsa, gendsa and dsaparam have been
-   deprecated.  Instead use the pkeyparam, pkey, genpkey and pkeyparam
-   programs respectively.
+   modified to use PKEY APIs in anticipation of their later deprecation.
```

Comment:
I would drop the `in anticipation of their later deprecation` and the following sentence so we do not introduce a fourth state (anticipation of deprecation) in addition to the already existing (current, deprecated, removed). :smile:

If we decided that these commands should not be deprecated, there is no point to warn that we might change our minds in distant future because that we might change our minds at some point and deprecate something applies practically to everything.

Reply:
Alternatively we could say something like that these commands are in maintenance mode with no new features being added to them.

Reply:
I prefer the maintenance mode option.